src/components/molecules/Common/AssetModal/AssetCard/index.tsx

```diff
@@ -10,7 +10,10 @@ type CardSize = "small" | "medium" | "large";
 export type Props = {
   className?: string;
   name: string;
-  url: string;
+  url?: string;
+  icon?: string;
+  iconSize?: string;
+  isImage?: boolean;
```

Comment:
Do you need the isImage? If you don't, it must have crept back during your merge.
@lavalse removed it in favor of the local regex expression since I believe the AssetContainer (which uses the  AssetCard) is being used now in the Settings page with ALL assets coming in. Which means each url needs to be checked locally to assign an icon or not.
Sorry, I know it might mean a bit of re-refactoring.

Reply:
yes, sorry for removing "isImage". but which is necessary , like the asset is image or not image , but they will be show in setting page at same time. if any difficult let me know